cmd/cosign/cli/verify.go

```diff
@@ -218,43 +218,43 @@ You may specify either a key, a certificate or a kms reference to verify against
 
 The signature may be specified as a path to a file or a base64 encoded string.
 The blob may be specified as a path to a file or - for stdin.`,
-		Example: `  cosign verify-blob (--key <key path>|<key url>|<kms uri>)|(--cert <cert>) --signature <sig> <blob>
+		Example: ` COSIGN_EXPERIMENTAL=1 cosign verify-blob (--key <key path>|<key url>|<kms uri>)|(--cert <cert>) --signature <sig> <blob>
 
   # Verify a simple blob and message
-  cosign verify-blob --key cosign.pub --signature sig msg
+  COSIGN_EXPERIMENTAL=1 cosign verify-blob --key cosign.pub --signature sig msg
 
   # Verify a simple blob with remote signature URL, both http and https schemes are supported
-  cosign verify-blob --key cosign.pub --signature http://host/my.sig
+  COSIGN_EXPERIMENTAL=1 cosign verify-blob --key cosign.pub --signature http://host/my.sig
 
   # Verify a signature from an environment variable
```

Comment:
let's axe this example? it's just explaining how the environment variables work in a shell, I think